editor/editor_about.cpp

```diff
@@ -64,7 +67,13 @@ void EditorAbout::_license_tree_selected() {
 	_tpl_text->set_text(selected->get_metadata(0));
 }
 
+void EditorAbout::_version_button_pressed() {
+	DisplayServer::get_singleton()->clipboard_set(version_btn->get_meta(META_TEXT_TO_COPY));
+}
+
 void EditorAbout::_bind_methods() {
+	ClassDB::bind_method("_version_button_pressed", &EditorAbout::_version_button_pressed);
+	ClassDB::bind_method(D_METHOD("_license_tree_selected"), &EditorAbout::_license_tree_selected);
```

Comment:
Any particular reason why one binding has D_METHOD and the other doesn't?

Reply:
I didn't notice this during the rebase; fixed it.